code/modules/organs/organ_external.dm

```diff
@@ -727,12 +728,16 @@ Note that amputating the affected organ does in fact remove the infection from t
 					"<span class='moderate'><b>Your [src.name] goes flying off!</b></span>",\
 					"<span class='danger'>You hear a terrible sound of [gore_sound].</span>")
 		if(DROPLIMB_BURN)
+			if(cannot_gib)
```

Comment:
Would it be better to do "if a and !b" rather than "if a, if b return" or is there something I'm missing here?